index.js

```diff
@@ -447,12 +450,7 @@ function checkPath (fastify, rootPath) {
 const supportedEncodings = ['br', 'gzip', 'deflate']
 
 function getContentType (path) {
-  const type = send.mime.lookup(path)
-  const charset = send.mime.charsets.lookup(type)
-  if (!charset) {
-    return type
-  }
-  return `${type}; charset=${charset}`
+  return send.mime.getType(path)
```

Comment:
I think htis has to be:

```suggestion
  const type = send.mime.getType(path)
 
  if (!send.isUtf8Mimetype(type)) {
    return type
  }
  return `${type}; charset=UTF-8`
```